lib/valkyrie/storage_adapter.rb

```diff
@@ -10,20 +10,23 @@ class << self
       # @param short_name [Symbol]
       # @return [void]
       def register(storage_adapter, short_name)
-        storage_adapters[short_name] = storage_adapter
+        storage_adapters[short_name.to_sym] = storage_adapter
```

Comment:
Our documentation says `short_name` should be a symbol.  Why should we then call `Symbol#to_sym`?

Reply:
I agree with @jcoyne here, but want to merge, so I pushed a commit to this PR for merging. Hope that's okay, @jeremyf